spot-client/src/spot-remote/ui/views/join-code-entry.js

```diff
@@ -76,31 +78,13 @@ export class JoinCodeEntry extends React.Component {
             code = queryParams.get('code');
         }
 
-        // Hide the code and other params for visual clarity only, no practical
-        // purpose.
-        this.props.history.replace(this.props.location.pathname);
-
         if (code) {
             logger.log('joinCodeEntry detected a valid code');
 
             this._connectToSpot(code);
         }
     }
 
-    /**
-     * Redirects to the remote control view when a connection to Spot is
-     * established.
-     *
-     * @inheritdoc
-     */
-    componentDidUpdate(prevProps) {
-        if (!prevProps.isConnectedToSpot && this.props.isConnectedToSpot) {
```

Comment:
what happened to this logic ? is it no longer relevant ?

Reply:
Yeah, while testing it looked dead and duplicated in another method in this file. I'll double check.

Reply:
Yeah, this block doesn't get fired on the normal case. I'm not too sure when this block was needed, but right now on join code submit there is already a redirect that happens to  ROUTES.REMOTE_CONTROL or now ROUTES.SHARE. I'm assuming when join code validation was mocked out, and the redirect logic added to the validation, that this block become unnecessary.